zebra-network/src/peer/handshake.rs

```diff
@@ -374,6 +392,23 @@ where
                         msg
                     }
                 })
+                .then(move |msg| {
+                    let inv_collector = inv_collector.clone();
+                    async move {
+                        if let Ok(Message::Inv(hashes)) = &msg {
+                            // We reject inventory messages with more than one
```

Comment:
The messages are used elsewhere in Zebra, so it would be clearer to say "ignore":

```suggestion
                            // We ignore inventory messages with more than one
```